.github/workflows/validate.yaml

```diff
@@ -0,0 +1,112 @@
+name: VALIDATE
+
+on:
+  push:
```

Comment:
Do we really want to trigger this for every brunch?

Reply:
I would say so. I also want to have the checks during development. Fixing errors continuously is easier than fixing a large chunk at the end.

Reply:
ok, I would just open an PR then, but ok, we can always disable if it becomes too much.